doc/manual_rest.rst

```diff
@@ -105,6 +105,7 @@ command:
           --stdin-filename filename                filename to use when reading from stdin (default "stdin")
           --tag tag                                add a tag for the new snapshot (can be specified multiple times)
           --time time                              time of the backup (ex. '2012-11-01 22:08:41') (default: now)
+          --use-windows-vss                        use windows volume shadow copy service
```

Comment:
`windows volume shadow copy service` -> `Windows Volume Shadow Copy Service`

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/3a3a297fe427caa8a2b918b62b4c8a70185d0483

Reply:
The description in `cmd_backup.go` is still in all lower-case.

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/738b8b8d2ba10be2f9f5f74128a9d69df19f0a05